pkg/controller/installation/core_controller.go

```diff
@@ -793,7 +801,7 @@ func mergeProvider(cr *operator.Installation, provider operator.Provider) error
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.V(1).Info("Reconciling Installation.operator.tigera.io")
+	reqLogger.Info("Reconciling Installation.operator.tigera.io")
```

Comment:
You're sure this isn't too spammy now? :grin: 

In general, we should be aiming for no repetitive log output. In an ideal world, the INFO level logs would only show when something meaningful has happened.

Reply:
:grin: 

From my tests, the log behaviour is much more sane now. At steady state, I only see the periodic reconcile invocations. I figured I would re-enable this since I believe it's the only controller that doesn't log when it reconciles, which can lead to misleading log interpretations.